### PR TITLE
First version of autocomplete for tagging in database collection settings

### PR DESCRIPTION
**Pull request by PineappleIOnic**
+ Autocomplete in tags
+ Remove webkit transform in select css

### CODE REVIEW

File: public/scripts/views/forms/tags.js [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
   window.ls.container.get("view").add({
     selector: "data-forms-tags",
-    controller: function(element) {
+    controller: function (element, sdk) {
       let array = [];
       let tags = window.document.createElement("div");
       let preview = window.document.createElement("ul");
+      let addContainer = window.document.createElement("div");
       let add = window.document.createElement("input");
+      let recomendations = { users: [], teams: [] };
```
Comment (TorstenDittmann) [inline public/scripts/views/forms/tags.js:12]: I guess you mean `recommendations` :wink:

File: public/scripts/views/forms/tags.js [hunk 2]
```diff
@@ -37,34 +48,103 @@
...
         return false;
       };
-      
-      let check = function() {
-        try {
-          array = JSON.parse(element.value) || [];
-        } catch (error) {
-          array = [];
+
+      let rerenderRecomendations = function () {
```
Comment (TorstenDittmann) [inline public/scripts/views/forms/tags.js:56]: Also here `rerenderRecommendations` :slightly_smiling_face:

**Review by TorstenDittmann — COMMENTED** [review]
Just a brief code review, looks fine so far. Gonna test it in action later :+1:

**Comment by abhee4** [comment]
Hello @TorstenDittmann and @PineappleIOnic ,

This is still not merged right ?

Is there a way I can help testing in beta or something ?

**Comment by PineappleIOnic** [comment]
Currently waiting on @eldadfux and @punitdiwan to review the code so it can be merged, but feel free to test the branch and let me know if you encounter any bugs with it 👍

**Comment by eldadfux** [comment]
Closing for now.

**Comment by gewenyu99** [comment]
@eldadfux @PineappleIOnic Do you mind filling in the context in the PR or in the issue for why this issue was closed?

**Pull request closed without merging**